lighthouse-core/gather/gather-runner.js

```diff
@@ -356,10 +356,10 @@ class GatherRunner {
               const passName = config.passName || Audit.DEFAULT_PASS;
               if (config.recordTrace) {
                 tracingData.traces[passName] = passData.trace;
-                tracingData.devtoolsLogs[passName] = passData.devtoolsLog;
               }
-              config.recordNetwork &&
-                  (tracingData.networkRecords[passName] = passData.networkRecords);
+
+              tracingData.devtoolsLogs[passName] = passData.devtoolsLog;
```

Comment:
this has the same problem as #2062 in that there's no `passData.devtoolLog` if there was no trace. Leave inside the conditional until that's fixed?